package.json

```diff
@@ -58,6 +58,7 @@
     "libsvm-js": "^0.2.1",
     "node-addon-api": "^1.3.0",
     "openseadragon": "^2.4.0",
+    "ramda": "^0.26.1",
```

Comment:
what is ramda for?

Reply:
I can't see it used anywhere...

Reply:
ah yeah, I mentioned that on slack. Ramda is a library of "functional" functions, that I used to rewrite the existing implementation so I could understand what it was doing. By the time I'd finished, I'd removed all the calls to Ramda again.

Reply:
ahh, sorry, didn't see that on slack.